committee-code-of-conduct/charter.md

```diff
@@ -65,7 +65,8 @@ authority and enforcement to this committee. The committee can, at its
 discretion, delegate some authority to those tasked with enforcement.*
 
 ## Election
-TODO: Our election processes will be outlined in a separate document, coming soon.
+
+See [elections](https://github.com/kubernetes/community/blob/master/committee-code-of-conduct/election.md)
```

Comment:
I think in this repo relative links are preferred, eg:

```See [elections](election.md)```

although there is a lot of inconsistency too.